docs/section12/Rule12-6.md

```diff
@@ -18,23 +18,12 @@
-- Get the receptacle load of each space in the building segment in the User model: ```For building_segment in U_RMR.building.building_segments:```  
-  - Get thermal_block from building segment: ```thermal_block in building_segment.thermal_blocks:```
-  - Get thermal_zone from thermal block: ```thermal_zone in thermal_block.zones:```
-  - Get space from thermal zone: ```space in thermal_zone.spaces:```  
-    - Get the name of the space: ```space_name = space.name```
-    - Get the receptacle load in the space: ```space_receptacle = space.equipment_internal_gains```
-      - Get the receptacle load schedule: ```receptacle_sch = space_receptacle.schedule_name```
-      - Save the receptacle load schedule for all spaces in the User model: ```space_receptacle_schedules_user[space_name] = receptacle_sch```
-
-- Get the receptacle load of each space in the building segment in the Proposed model: ```For building_segment in P_RMR.building.building_segments:```  
-  - Get thermal_block from building segment: ```thermal_block in building_segment.thermal_blocks:```
-  - Get thermal_zone from thermal block: ```thermal_zone in thermal_block.zones:```
-  - Get space from thermal zone: ```space in thermal_zone.spaces:```  
-    - Get the name of the space: ```space_name = space.name```
-    - Get the receptacle load in the space: ```space_receptacle = space.equipment_internal_gains```
-      - Get the receptacle load schedule: ```receptacle_sch = space_receptacle.schedule_name```
-      - Save the receptacle load schedule for all spaces in the Proposed model: ```space_receptacle_schedules_proposed[space_name] = receptacle_sch```
-
-**Rule Assertion:** ```space_receptacle_schedules_user == space_receptacle_schedules_proposed```  
+- **Applicability Check 1:** ```if (rule-12-3.status == TRUE):```
+- Get the receptacle load of each space in the building segment in the User model: ```for building_segment_user in U_RMR.building.building_segments:```  
+  - For each thermal_block in building segment: ```thermal_block_user in building_segment_user.thermal_blocks:```
+  - For each zone in thermal block: ```zone_user in thermal_block_user.zones:```
+  - For each space in thermal zone: ```space_user in zone_user.spaces:```  
+    - For each miscellaneous equipment in the space, get the equipment schedule: ```for space_equipment_user in space_user.miscellaneous_equipments: equipment_sch_user_dict[space_equipment_user] = space_equipment_user.schedule_name```
+    - Get matching space from Proposed RMR: ```space_proposed = match_data_element(P_RMR, spaces, space_user.name)```
+      - For each miscellaneous equipment in the space, get the equipment schedule: ```for space_equipment_proposed in space_proposed.miscellaneous_equipments: equipment_sch_proposed_dict[space_equipment_proposed] = space_equipment_proposed.schedule_name```
+    **Rule Assertion:** ```equipment_sch_user_dict == equipment_sch_proposed_dict```  
```

Comment:
Small formatting issue, please add blank line before rule assertion line so it is by itself.

Reply:
Thanks. Updated in the file.

Reply:
Original discussion at the link below. Please give this link time to load. This page might take a few seconds to load and another 4-8 seconds to land at the correct comment:

https://github.com/pnnl/ruleset-checking-tool/pull/19#discussion_r614294064

Reply:
@xingcx can we use the schedule comparison function here to compare the U and P receptacle schedule?